src/OpenLoco/src/Vehicles/VehicleTail.cpp

```diff
@@ -137,7 +137,7 @@ namespace OpenLoco::Vehicles
                 tad._data = routing & Track::AdditionalTaDFlags::basicTaDMask;
                 tail.sub_47D959(pos, tad, false);
 
-                pos += World::TrackData::getUnkRoad(tad._data).pos;
+                pos += World::TrackData::getUnkRoad(tad._data & 0x7F).pos;
```

Comment:
Accessor function on the TaD data type?

Reply:
I'm not sure how best to approach these. I'd like to really rework a lot of the tad related things but I'm not quite sure what that would end up looking like.